Update troubleshooting.md with troubleshooting details for containers. (#2275)
ollama/ollama: docs/troubleshooting.md

> 
> Join the [Discord](https://discord.gg/ollama) for help interpreting the logs.
> 
> In a container running on docker, the logs can be found by calling `docker logs <container-name>`
> Frequently:
> ```shell
> docker logs ollama
> ```
> 

Reviewer: ```suggestion

If using Docker, logs can be found by running `docker logs ollama`:
```shell
docker logs ollama
```

Join the [Discord](https://discord.gg/ollama) for help interpreting the logs.
```
